main.rb

```diff
@@ -0,0 +1,25 @@
+require './my_enumerable'
+
+class MyList
+  def initialize(*item)
+    @list = item
+  end
+  include MyEnumerable
+  def each(&block)
+    @list.each(&block) if block_given?
+  end
+end
+
+# Create our list
+list = MyList.new(1, 2, 3, 4)
+
+# Test #all?
+list.all? { |e| e < 5 }
+list.all? { |e| e > 5 }
+
+# Test #any?
+list.any? { |e| e == 2 }
+list.any? { |e| e == 5 }
+
+# Test #filter
+list.filter(&:even?)
```

Comment:
# :clap: :clap: 
Hi  @mperezbazan  & @emmanuellmhango good job, your application is working as expected.

![Screenshot from 2023-01-27 01-01-30](https://user-images.githubusercontent.com/16392046/214960725-69e98cb8-628e-4d19-aea2-a614de7bc855.png)